apps/zui/src/domain/editor/parse.test.ts

```diff
@@ -3,12 +3,19 @@
  */
 import {SystemTest} from "src/test/system"
 import {parse} from "./operations"
+import {ZedAst} from "src/app/core/models/zed-ast"
 
 new SystemTest("editor.parse op")
 
 test("editor.parse", async () => {
   const result = await parse("from source | count() by name")
-  expect(result.map((o) => o.kind)).toEqual(["From", "Summarize"])
+  expect(result.map((o) => o.kind)).toEqual(["Pool", "Summarize"])
+})
+
+test("Zed Ast Pool Name", async () => {
+  const result = await parse("from source | count() by name")
```

Comment:
Another good test here would be to test that it successfully gets the name from a multi-from:
```
from (
  pool test1
  pool test2
)```